ingestion/src/metadata/mixins/pandas/pandas_mixin.py

```diff
@@ -80,20 +80,23 @@ def get_partitioned_df(self, dfs):
             for df in dfs
         ]
 
-    def return_ometa_dataframes_sampled(  # pylint: disable=inconsistent-return-statements
+    def return_ometa_dataframes_sampled(
         self, service_connection_config, client, table, profile_sample_config
     ):
         """
         returns sampled ometa dataframes
         """
+        from pandas import DataFrame  # pylint: disable=import-outside-toplevel
 
         data = ometa_to_dataframe(
             config_source=service_connection_config.configSource,
             client=client,
             table=table,
         )
+        if isinstance(data, DataFrame):
+            data: List[DataFrame] = [data]
 
-        if data:
+        if data and isinstance(data, list):
```

Comment:
why would `data` not be a list? Shouldn't `ometa_to_dataframe` always return a list of dataframes?

Reply:
Yes, this is just a safeguard statement (maybe an overkill for a safeguard) so that things don't fail.

Reply:
🤔 I think here we would not be handling things properly right and make it a bit harder to debug. If we don't have a list of dataframes, if we want to keep that `if` logic, then let's raise an error and not return an empty list.

Reply:
we are handling the return of list of dataframes right for now. We have a ticket open to improve the logic for all the connections to be taken care of in 1.1.
let me add a warning for dataframe names which are not returned as list if there is an edge case by any chance and raise the error.